Tests/sample.ps1

```diff
@@ -0,0 +1,18 @@
+$ResourceGroupName = "dfc-test-template-rg"
+$TemplateFile = "$PSScriptRoot\..\ArmTemplates\keyvault.json"
+$params = @{ keyVaultName = "dfc-foo-bar-kv" }
+
+$output = Test-AzureRmResourceGroupDeployment `
+              -ResourceGroupName $ResourceGroupName `
+              -TemplateFile $TemplateFile `
+              -TemplateParameterObject $params `
+              -ErrorAction Stop `
+              5>&1

```

Comment:
Not sure what this does

Reply:
sample.ps1 should have been deleted. I was using it for debugging - will fix.